From 49da22781d7bf8b6e43f44e56eb96227e7d081c4 Mon Sep 17 00:00:00 2001
From: Davis Goodin <dagood@microsoft.com>
Date: Wed, 4 Nov 2020 14:03:23 -0600
Subject: [PATCH] Remove unused Microsoft.Win32.Registry/4.6.0 ref

This project restores a prebuilt Microsoft.Win32.Registry reference, but
it doesn't end up in the SDK: the DLL from the net5.0 shared framework
is already present. Remove this unnecessary PackageReference to remove
the prebuilt.
---
 src/Cli/dotnet/dotnet.csproj | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/Cli/dotnet/dotnet.csproj b/src/Cli/dotnet/dotnet.csproj
index c6a20563e9..babd0423ac 100644
--- a/src/Cli/dotnet/dotnet.csproj
+++ b/src/Cli/dotnet/dotnet.csproj
@@ -74,7 +74,6 @@
     <PackageReference Include="Microsoft.TemplateEngine.Orchestrator.RunnableProjects" Version="$(MicrosoftTemplateEngineOrchestratorRunnableProjectsPackageVersion)" />
     <PackageReference Include="Microsoft.TemplateEngine.Utils" Version="$(MicrosoftTemplateEngineUtilsPackageVersion)" />
     <PackageReference Include="Microsoft.TemplateSearch.Common" Version="$(MicrosoftTemplateSearchCommonPackageVersion)" />
-    <PackageReference Include="Microsoft.Win32.Registry" Version="4.6.0" />
   </ItemGroup>
   <ItemGroup Condition=" '$(IncludeAspNetCoreRuntime)' != 'false' ">
     <PackageReference Include="Microsoft.AspNetCore.DeveloperCertificates.XPlat" Version="$(MicrosoftAspNetCoreDeveloperCertificatesXPlatPackageVersion)" />
-- 
2.25.2

